core-bundle/config/listener.yaml

```
@@ -188,19 +188,25 @@ services:
             - '@contao.twig.loader.theme_namespace'
             - '@translator'
 
-    contao.listener.data_container.undo.jump_to_parent_button:
-        class: Contao\CoreBundle\EventListener\DataContainer\Undo\JumpToParentButtonListener
+    contao.listener.data_container.undo.jump_to_parent_operation:
+        class: Contao\CoreBundle\EventListener\DataContainer\Undo\JumpToParentOperationListener
         arguments:
             - '@contao.framework'
             - '@database_connection'
             - '@translator'
+            - '@security.helper'
 
     contao.listener.data_container.undo.label:
         class: Contao\CoreBundle\EventListener\DataContainer\Undo\LabelListener
         arguments:
             - '@contao.framework'
             - '@twig'
 
+    contao.listener.data_container.undo_operation:
+        class: Contao\CoreBundle\EventListener\DataContainer\Undo\UndoOperationListener
```

```suggestion
        class: Contao\CoreBundle\EventListener\DataContainer\Undo\OperationListener
```

At least we didn‘t call the `Contao\CoreBundle\EventListener\DataContainer\Undo\LabelListener` class `UndoLabelListener`.

---

yeah but the `JumpToParentOperationListener` exists as well. `Undo` in `UndoOperation` represents the operation name, not the table.